h1st/model_repository/model_repository.py

```diff
@@ -80,17 +80,17 @@ def serialize(self, model, path):
         """
         meta_info = {}
 
-        if hasattr(model, 'metrics'):
+        if model.metrics:
             logger.info('Saving metrics property...')
```

Comment:
You're not `saving metrics property`, you're `persisting model metrics` or `saving model metrics`

Reply:
Be very considerate about these core system outputs.